tests/datadriven/__init__.py

```diff
@@ -194,6 +200,24 @@ def assertRaisesWith(self, exceptionType):
                 "exception of type {} not raised".format(
                     exceptionType.__name__))
 
+    def assertAlmostEqual(self, a, b, ndigits=7):
+        """
+        Assert that a and b are equal within ndigits digits
+        """
+        if round(a-b, ndigits) != 0:
```

Comment:
@dcolligan perhaps I mis-phrased my question. Do we want a `assertAlmostEqual` to test a comparison of two floats to within n significant digits, or n decimal points? 
I would argue for the former, but welcome feedback on the matter.

Consider:
```
>>> a = 1.11111111E20
>>> b = 1.11111112E20
>>> round(a-b, 5)
-999999995904.0
```